src/components/user/IFXUserDetail.vue

```diff
@@ -96,13 +96,30 @@ export default {
             </v-col>
           </v-row>
           <v-row dense>
-            <v-col xs="4" md="3">
+            <v-col xs="4" md="3" v-if="additionalContactList()">
               <h3>Other Contacts</h3>
             </v-col>
             <v-col>
               <span v-html="additionalContactList()"></span>
             </v-col>
           </v-row>
+          <v-row dense v-if="(item.accounts && item.accounts.length) || (item.productAccounts && item.productAccounts.length)">
```

Comment:
Is it worth changing these to use `?.` also?

Reply:
Done